build.sh

```diff
@@ -99,4 +99,4 @@ fi
 
 # Build EOS
 cmake -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DCMAKE_CXX_COMPILER=${CXX_COMPILER} -DCMAKE_C_COMPILER=${C_COMPILER} -DWASM_LLVM_CONFIG=${WASM_LLVM_CONFIG} -DBINARYEN_BIN=${BINARYEN_BIN} -DOPENSSL_ROOT_DIR=${OPENSSL_ROOT_DIR} -DOPENSSL_LIBRARIES=${OPENSSL_LIBRARIES} ..
-make -j4
+make -j$(nproc)
```

Comment:
For Mac Os this command is invalid 
```sysctl -n hw.physicalcpu```
So , it has to be some logic if/else in this script